meta-lmp-base/classes/lmp.bbclass

```diff
@@ -125,10 +125,18 @@ IMAGE_CMD:ota:append () {
 	if [ "${APP_PRELOAD_WITHIN_OE_BUILD}" = "1" ]; then
 		preload_apps
 	fi
+
+	# Split content /var into a separated folder so it can be consumed by WKS separately
+	if [ "${OSTREE_SPLIT_VAR}" = "1" ]; then
+		rm -rf ${OTA_VAR}
+		mv ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var ${OTA_VAR}
+		mkdir -p ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var
```

Comment:
Using `mv` will fail if we run this task more than one time because on the second one the `${OSTREE_OSNAME}/var` is clean. It should be better to copy all the files with `cp -a` and we can drop the `mkdir`.

Reply:
We don't need to have the content in `${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var` after this code.

But I agree we may need to move the content only once like
```
if [ "${OSTREE_SPLIT_VAR}" = "1" ] && ! -d ${OTA_VAR}; then
		mv ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var ${OTA_VAR}
		mkdir -p ${OTA_SYSROOT}/ostree/deploy/${OSTREE_OSNAME}/var
fi
```

Reply:
OTA_SYSROOT is already defined for dir and cleandirs:
```
do_image_ota[dirs] = "${OTA_SYSROOT}"
do_image_ota[cleandirs] = "${OTA_SYSROOT}"
```

So a re-run wouldn't be an issue here.

mkdir might not necessarily be needed indeed, will check.